src/Coordination/Changelog.h

```diff
@@ -144,7 +152,9 @@ class Changelog
     /// Mapping log_id -> log_entry
```

Comment:
Related to line above, just to understand:
Is `existing_changelogs` keys is from_log_index from corresponding values?

Reply:
Yes.